apps/src/lab2/responseValidators.ts

```diff
@@ -1,32 +1,79 @@
 import {ResponseValidator} from '@cdo/apps/util/HttpClient';
-import {BlocklySource, LevelProperties, ProjectSources} from './types';
+import {
+  BlocklySource,
+  LevelProperties,
+  NestedSourceCode,
+  ProjectSources,
+} from './types';
+import Lab2Registry from './Lab2Registry';
+import {BLOCKLY_LABS} from './constants';
 
-export const SourceResponseValidator: ResponseValidator<
+// Validator for Blockly sources.
+export const BlocklySourceResponseValidator: ResponseValidator<
```

Comment:
These no longer need to be exported right?

Reply:
yep, good catch